apps/browser/qml/pages/components/AddHomePageDialog.qml

```diff
@@ -37,7 +38,7 @@ Dialog {
         label: placeholderText
         inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhNoAutoUppercase | Qt.ImhUrlCharactersOnly
         EnterKey.iconSource: "image://theme/icon-m-enter-close"
```

Comment:
This should be the normal enter key. Afair by heart, enter-close points to downwards. Removing this should be enough.